### PR TITLE
Mermaidjs translation

### PR DESCRIPTION
**Pull request by arvidbt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,4 +219,10 @@ To parse an dataset, the CSV file must contain a column which is named `Model JS
     "LTLf": "F((accept_invoice) & (X[!](false)))"
   }
 ]
+```
+3. Translating the diagram to Mermaid.js.
+```yaml
```
Comment (TimKam) [inline README.md:224]: Doesn't it make sense to have start and end events here? Also, do we really need the label 'check invoice' twice? Can't we just use the ID?

File: bpmnconstraints/mermaid/mermaidtranslation.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
+from bpmnconstraints.utils.constants import *
+
+DEFAULT_DIRECTION = "LR"
+SEQUENCE_FLOW = "-->"
+END_EVENT_STYLING_DEF = "classDef EndEvent fill:stroke:#000,stroke-width:4px"
+END_EVENT_STYLE = ":::EndEvent"
+
+
+class Mermaid:
```
Comment (TimKam) [inline bpmnconstraints/mermaid/mermaidtranslation.py:9]: Consider adding some docstrings here. I suppose in-code documentation wasn't a prio in general so far...

**Review by TimKam — CHANGES_REQUESTED** [review]
Very neat! I added some comments. Also, can you add 1-3 tests, simply based on existing examples to control for potential future regressions? Afterwards, you can merge.

File: README.md (hunk 1 above)
Comment (arvidbt) [reply README.md:224]: As in startnoneevent etc? As I'm just basing this on the parser, it deems that the 'register invoice' is a better start event than the startnoneevent.

Comment (arvidbt) [reply README.md:224]: Added issues for this.

**Pull request merged**